### PR TITLE
Add an option to disable Passkeys

### PR DESCRIPTION
**Pull request by kean**
This PR adds a temporary option to disable passkeys.

To test: see https://github.com/wordpress-mobile/WordPress-iOS/pull/22113. 

---

- [ ] I have considered if this change warrants release notes and have added them to the appropriate section in the `CHANGELOG.md` if necessary.

### CODE REVIEW

File: WordPressAuthenticator/Authenticator/WordPressAuthenticatorConfiguration.swift [hunk 1]
```diff
@@ -190,6 +193,7 @@ public struct WordPressAuthenticatorConfiguration {
                  enableSignupWithGoogle: Bool = false,
                  enableUnifiedAuth: Bool = false,
                  enableUnifiedCarousel: Bool = false,
+                 enablePasskeys: Bool = true,
```
Comment (mokagio) [inline WordPressAuthenticator/Authenticator/WordPressAuthenticatorConfiguration.swift:196]: Thank you for adding a default value so this remain backward compatible.

**Review by mokagio — COMMENTED** [review]
Straightforward change 👍 

Was unsure how to test this manually, so I wrote some unit tests for it https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/810

**Review by momo-ozawa — APPROVED** [review]
Tested via https://github.com/wordpress-mobile/WordPress-iOS/pull/22113, works as described

**Comment by momo-ozawa** [comment]
@Ecarrion - just a heads up, no changes required on the Woo side

**Comment by kean** [comment]
I was originally planning to merge this in trunk and release as 7.4.0, but I noticed there have been multiple changes (refactoring) made in [trunk](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/commits/trunk) since the previous release. I would suggest erring on the side of caution and either releasing this as a patch for 7.3.0 or merging the original PR in WordPress-iOS https://github.com/wordpress-mobile/WordPress-iOS/pull/22113 with this branch as a dependency. @mokagio, I would appreciate your help in deciding the best way forward.

**Comment by mokagio** [comment]
> I would suggest erring on the side of caution and either releasing this as a patch for 7.3.0 or merging the original PR in WordPress-iOS https://github.com/wordpress-mobile/WordPress-iOS/pull/22113 with this branch as a dependency.

Good thinking @kean. For the sake of keeping releases tidy, I'll take this over and ship it as 7.3.1. 🙇‍♂️

**Comment by mokagio** [comment]
Closing in favor of #812.

Next steps:

- Deploy 7.3.1
- Point https://github.com/wordpress-mobile/WordPress-iOS/pull/22113 to 7.3.1

**Pull request closed without merging**